src/js/editionSwitch.js

```diff
@@ -0,0 +1,61 @@
+import utils from './utils';
+
+const defaultClassName = 'o-header__edition-switch';
+class EditionSwitcher {
```

Comment:
Is there a reason this is titled `EditionSwitcher` and everything else that references it (filename as well) is `EditionSwitch`?